shell/platform/linux/fl_view.cc

```diff
@@ -155,9 +167,12 @@ static void fl_view_constructed(GObject* object) {
   GdkDisplay* display = gtk_widget_get_display(GTK_WIDGET(self));
   self->renderer = fl_view_get_renderer_for_display(display);
   self->engine = fl_engine_new(self->project, self->renderer);
+  fl_engine_set_update_semantics_node_handler(
+      self->engine, fl_view_update_semantics_node_cb, self, nullptr);
```

Comment:
Do we need a corresponding nulling out of this callback in the view's destructor?